File: csharp/DVSClient/DVSClient/address/format/components/AdditionalSubRegion.cs
using DVSClient.Server.Address.Format;

namespace DVSClient.Address.Format.Components
{
    public class AdditionalSubRegion
    {
        public string AdministrativeCounty { get; }
        public string FormerPostalCounty { get; }
        public string TraditionalCounty { get; }

        public AdditionalSubRegion(RestApiAddressFormatComponents.RestApiAddressComponentAdditionalSubRegion api)
        {
            AdministrativeCounty = api.AdministrativeCounty ?? string.Empty;
            FormerPostalCounty = api.FormerPostalCounty ?? string.Empty;
            TraditionalCounty = api.TraditionalCounty ?? string.Empty;
        }
    }
}

File: java/src/main/java/com/experian/dvs/client/address/format/components/AdditionalSubRegion.java
package com.experian.dvs.client.address.format.components;

import com.experian.dvs.client.server.address.format.RestApiAddressFormatComponents;

import java.util.Objects;

public class AdditionalSubRegion {
    private final String administrativeCounty;
    private final String formerPostalCounty;
    private final String traditionalCounty;

    // Constructor
    public AdditionalSubRegion(RestApiAddressFormatComponents.RestApiAddressComponentAdditionalSubRegion api) {
        this.administrativeCounty = Objects.toString(api.getAdministrativeCounty(), "");
        this.formerPostalCounty = Objects.toString(api.getFormerPostalCounty(), "");
        this.traditionalCounty = Objects.toString(api.getTraditionalCounty(), "");
    }

    // Getters
    public String getAdministrativeCounty() {
        return administrativeCounty;
    }

    public String getFormerPostalCounty() {
        return formerPostalCounty;
    }

    public String getTraditionalCounty() {
        return traditionalCounty;
    }
}
